app/controllers/subscriptions_controller.rb

```diff
@@ -0,0 +1,28 @@
+class SubscriptionsController < AuthenticatedController
+  include ProjectScoped
+
+  def create
+    subscription = Subscription.new(subscription_params)
+    subscription.user = current_user
+    subscription.save
+
+    redirect_to [@project, subscription.subscribable], notice: 'Subscribed!'
```

Comment:
Probably add more info to the `notice` like `Subscribed to Issue SSL Vulnerability` etc. Same with unsubscribe.

Reply:
We are exactly viewing the item in question, let's to move on. 
In fact I am not sure of adding a :notice at all.